dds/idl/dds_generator.cpp

```diff
@@ -348,7 +348,7 @@ string type_to_default(const std::string& indent, AST_Type* type, const string&
     AST_Enum* enu = dynamic_cast<AST_Enum*>(actual_type);
     UTL_ScopeActiveIterator i(enu, UTL_Scope::IK_decls);
     AST_EnumVal *item = dynamic_cast<AST_EnumVal*>(i.item());
-    def_val = item->name()->get_string_copy();
+    def_val = item->name()->get_string();
```

Comment:
Okay, looks like I lead you astray. That's what I get for commenting without actually looking at the code. Now that I'm actually paying attention, you can use:

```suggestion
    def_val = scoped(item->name());
```

Reply:
Yes... also while we're here let's update the lines right after this so that it's not immediately overwritten in C++11-mode